runtime/compiler/runtime/RelocationRecord.cpp

```diff
@@ -2814,11 +2825,7 @@ TR_RelocationRecordInlinedMethod::inlinedSiteValid(TR_RelocationRuntime *reloRun
       cp = J9_CP_FROM_METHOD(callerMethod);
    RELO_LOG(reloRuntime->reloLogger(), 6, "\tinlinedSiteValid: cp %p\n", cp);
 
-   if (!cp)
```

Comment:
I won't insist that this must remain as it is, but I will explain why I don't like this change, and then you can decide whether you're convinced :sweat_smile:

Essentially, for reading, code like this:
```
if (...)
   {
   // much code
   }
else
   {
   // one line
   }
```
has worse locality than code in the opposite order:
```
if (!(...))
   {
   // one line
   }
else
   {
   // much code
   }
```
In either case the merge (at the end) is far from the branch, but if we put the long clause first, then the `else` clause is also far from the branch. If OTOH we put the short clause first (as I prefer), then at least the branching part of the control flow (including both possible targets) is localized within a few lines.

Reply:
> hehe

:grin: